validator/test/fixtures/gke_master_authorized_networks_enabled/constraints/enable_master_authorized_networks/data.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPGKEMasterAuthorizedNetworksEnabledConstraintV1
+metadata:
+  name: enable_gke_master_authorized_networks
+  annotations:
+    benchmark: GKE_HARDENING_GUIDELINE
+spec:
+  severity: high
+  match:
```

Comment:
and here I think we can probably delete the whole matching block, since it doesn't seem to add any value

Reply:
updated